src/pipeline.ipynb

```diff
@@ -42,8 +91,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Setup your environment \n",
-    "!pip install \"fondant==0.6.2\""
+    "try:\n",
```

Comment:
We can remove this cell if we're installing the `requirements.txt`. I did the same in the controlnet example.